CRM/Core/BAO/Cache.php

```diff
@@ -192,9 +192,14 @@ public static function cleanup($session = FALSE, $table = FALSE, $prevNext = FAL
     }
 
     if ($expired) {
+      $cache = CRM_Utils_Cache::singleton();
```

Comment:
@totten can you take a look at this PR and, in particular, why I can't instantiate the SqlGroup cache driver this way? I wanted to do it this way so I could remove expired items in FileCache but also wanted it to work for the other drivers. The default is `ArrayCache` but how do we get the items that are still stored in the db?

Reply:
I figured it out. Using `\Civi::cache('long')` will get the `SqlGroup` or the `CIVICRM_DB_CACHE_CLASS` specified.